symengine/cwrapper.h

```diff
@@ -39,6 +39,13 @@ typedef enum {
     SYMENGINE_TypeID_Count
 } TypeID;
 
+//! Struct to hold the real and imaginary parts of std::complex<double>
+//! extracted from basic
+typedef struct dcomplex {
+    double real;
+    double imag;
+} dcomplex;
```

Comment:
Instead of defining our own complex type, why cannot we use the C complex type?

http://en.cppreference.com/w/c/numeric/complex